### ISSUE TITLE
r_draw_wireframe = kaputt (HELP!)

### ISSUE THREAD

**Issue by Panjoo** [jdolan/quetoo#459]
For the last week or so I'm optimizing VIS on Gehenna and my other new map but now after I updated, r_draw_wireframe suddenly crashes my game. 

![crappygame](https://cloud.githubusercontent.com/assets/10857572/24323210/cff8075a-1172-11e7-9f18-69f0140c40ce.jpg)

And even more sad, the other new map doesn't want to load up at all anymore (map is still fullbright, so that might have to do with this.) 
So, what am I to do now? I need wireframe to work on VIS and my other map which was going so well is now crashing. There's steam coming out my ears because this is so frustrating. F*ck this, really.

I can't make maps when my tools keep breaking, I'm not updating again until I'm done with these maps.  I'm not cloning the entire game every time I want to run the updater.

**Comment by kaadmy** [comment]
I can confirm this, it's related to tints, probably due to moving some stuff around recently.
```
0x0000000000429303 in R_UpdateMeshTints () at r_mesh.c:202
202		if (r_mesh_state.material->tintmap) {
(gdb) bt
#0  0x0000000000429303 in R_UpdateMeshTints () at r_mesh.c:202
#1  0x00000000004294ac in R_SetMeshState_default (e=0x1d33ef8 <r_view+9720>) at r_mesh.c:247
#2  0x00000000004298e6 in R_DrawMeshModel_default (e=0x1d33ef8 <r_view+9720>) at r_mesh.c:361
#3  0x0000000000429b59 in R_DrawMeshModels_default (ents=0x16033e8 <r_sorted_entities+8200>) at r_mesh.c:422
#4  0x000000000044a2c0 in R_DrawEntities () at r_entity.c:280
#5  0x0000000000424223 in R_DrawView () at r_main.c:193
#6  0x000000000040c610 in Cl_UpdateScreen () at cl_screen.c:353
#7  0x0000000000408db5 in Cl_Frame (msec=13) at cl_main.c:638
#8  0x0000000000407458 in Frame (msec=13) at main.c:429
#9  0x000000000040765b in main (argc=1, argv=0x7fffffffea28) at main.c:478
```

**Comment by kaadmy** [comment]
Fixed in a772710fcb570deb61fbeab23e2fd059a9e05dba
